source/initHQ/fortify.sqf

```diff
@@ -1,4 +1,5 @@
 _centerPos = _this select 0;
+_hq = _this select 1;
```

Comment:
This is more for my own edification... `_hq = _this select 1;` takes the second parameter the script was executed with and places it into the `_hq` variable? I know that question doesn't inspire much from a reviewer but I'm still trying to learn :-)